jetson/percep/pcl.cpp

```diff
@@ -458,14 +366,15 @@ shared_ptr<pcl::visualization::PCLVisualizer> PCL::createRGBVisualizer() {
 //This function is called in main.cpp
 obstacle_return PCL::pcl_obstacle_detection(shared_ptr<pcl::visualization::PCLVisualizer> viewer) {
     obstacle_return result;
-    PassThroughFilter();
+    PassThroughFilter("z", 7000.0);
+    PassThroughFilter("y", 3000.0);
```

Comment:
Again, this was in another PR already which is an issue on it's own but to be safe see the comment in that PR for fixing these two lines.